english/marshmallow.md

```diff
@@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
 
 Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
 
-Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
```

Comment:
Hi Caroline,

Thanks for reviewing this.  I should have seen the note about white spaces in Casey’s reply.  I fixed the line length issue and pushed the edited prostate.md file to my repository.  Do I need to file a new pull request, or can you take it from there?

Best,

Jeff

> On Nov 2, 2015, at 9:55 AM, Caroline Buckey <notifications@github.com> wrote:
> 
> In english/marshmallow.md <https://github.com/udacity/create-your-own-adventure/pull/2685#discussion_r43658225>:
> 
> > @@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
> >  
> >  Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
> >  
> > -Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
> @bynumjj <https://github.com/bynumjj>, it looks like you removed a space from the end of a few lines. (This shows up as "deleting" the original line and adding a new line without the space.) My guess is your editor does this automatically. That is fine by me - in fact, I prefer it if there are no spaces at the end of lines!
> 
> If you break up the long line Casey mentioned below into multiple lines, I'll be happy to merge this. No need to fix the "deleted" lines.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/udacity/create-your-own-adventure/pull/2685/files#r43658225>.
>